cmd/credentialUtil.go

```diff
@@ -140,9 +140,10 @@ func getBlobCredentialType(ctx context.Context, blobResourceURL string, canBePub
 // 2. If there is OAuth token info passed from env var, indicating using token credential. (Note: this is only for testing)
 // 3. Otherwise use shared key.
 func getBlobFSCredentialType() (common.CredentialType, error) {
-	if oAuthTokenExists() {
-		return common.ECredentialType.OAuthToken(), nil
-	}
+	// TODO uncomment before merging
```

Comment:
Also leaving a comment here to remind you to cherrypick in the SAS ADLSG2 commit and make use of it in tests